schedule/autoyast_mini_no_product_staging.yaml

```diff
@@ -0,0 +1,21 @@
+---
+name:           autoyast_mini_no_product
+description:    >
+  Test verifies installation with minimal autoyast profile.
+vars:
+  AUTOYAST: autoyast/mini.xml
+  AUTOYAST_CONFIRM: 1
+  HDDMODEL: scsi-hd
+  HDDSIZEGB: 30
+schedule:
+  - installation/bootloader
```

Comment:
As I can see in 'autoyast_mini_staging' (e.g. here https://openqa.suse.de/tests/2809792), the test suite starts from autoyast/prepare_profile module.

Also,  'autoyast_mini_no_product sle12' link in PR description redirects to 'RAID1' scenario, so I don't see if the test suite passes. Could you please double check it.

Reply:
fixed the missing step. and updated the link to the autoyast_mini_no_product sle12